src/main/java/com/minecolonies/coremod/entity/ai/util/Structure.java

```diff
@@ -159,6 +161,13 @@ else if ((structureBlock == Blocks.DIRT || structureBlock == Blocks.GRASS) && (w
 
             return structureBlockState.equals(worldBlockState);
         }
+
+        private static boolean structureBlockEqualsWorldBlock(@NotNull final Block structureBlock,
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Remove this unused private "structureBlockEqualsWorldBlock" method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AUnusedPrivateMethod)